Nice-Array/app.js

```diff
@@ -0,0 +1,10 @@
+function isNice(arr){
+  if (arr.length === 0){
```

Comment:
Your patterns of curly braces here are very weird! I'd rather write this conditional as:
```
if (arr.length === 0) {
  return false;
}
```
And there's a similar issue below, at your for loop.